source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,122 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
+
+.. table:: 
+    :widths: 50 50
+
+    +-----------------------------+---------------------+----------------------------------------------+
+    |Field type                   |Encryption Algorithm | BSON Type                                    |
+    +=============================+=====================+==============================================+
+    | Name                        | Non-Encrypted       | String                                       |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | SSN                         | Deterministic       |  Int                                         |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Blood Type                  | Random              |  String                                      |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Medical Records             | Random              |  Array                                       |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Policy Number    | Deterministic       |  Int (embedded inside `insurance` object)    |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Provider         | Non-Encrypted       |  String (embedded inside `insurance` object) |
+    +-----------------------------+---------------------+----------------------------------------------+
+
+To automatically encrypt and decrypt create a JSON Schema using the information in the data model and pass the schema as a JSON object to the secure client to specify the encrypted field properties.
+In the JSON, you can specify `encryptMetadata <https://docs.mongodb.com/manual/reference/security-client-side-automatic-json-schema/#encryptmetadata-schema-keyword>`_ that is passed down to the fields under the ``properties``
+nested object. 
+
+.. code::
+
+        {
+            "bsonType" : "object",
+            "encryptMetadata" : {
+                "keyId" : // copy and paste your keyID generated here
+            },
+            "properties": {
+                // copy and paste your fields here 
+            }
+        }
+
+Create JSON objects for each field and append them to the `properties` map.
+
+.. code::
+
+    // Deterministic encryption supports querying and is equally as secure as random encryption for high cardinality (very unique) fields such as SSN. A privileged user in the MedcoMD Medical Care Management System might want to query on SSN to verify a patient's identity. One thing to note is that you do not have to include keyId if you specified it in the top-level encryptMetadata field. Though you could include it if you want different users with specific keys to be able to access a specific subset of the encrypted data.
```

Comment:
We should standardize `keyId` (whether `keyID`, `keyId`, or other). The last sentence is not complete -- perhaps you can fix it by removing "Though". See https://english.stackexchange.com/questions/120981/using-though-at-the-beginning-of-the-following-sentence

Reply:
Fixed. It now reads:
* "One thing to note is that you do not have to include keyId if you specified it in the top-level encryptMetadata field. You could include keyId if you want different users with specific keys to be able to access a specific subset of the encrypted data."